docs/running-a-node/wallet-setup.md

```diff
@@ -0,0 +1,200 @@
+
+!!! info "Node wallet not for production use"
+	The node wallet is not supported for use in production environments. Use third-party or custom key management for production applications.
+
+Below is a basic walkthrough on how to setup a wallet and make a transaction using the built-in node wallet. It is important to understand the differences between accounts, private and public keys, seeds and wallet IDs before going much further. If you aren't comfortable with these concepts yet, see [this guide](https://docs.nano.org/integration-guides/the-basics/#account-key-seed-and-wallet-ids). Creating a wallet with a seed and generating accounts from that seed will be the focus of this guide.
+
+### Update configuration
+
+A configuration update is required to complete this guide. If you aren't familiar with configuring the node, see the [configuration guide](../running-a-node/configuration.md) which includes [configuration file locations](../running-a-node/configuration.md#configuration-file-locations) for various operating systems and other useful details.
+
+--8<-- "config-node-option-rpc-enable-control-true.md"
+
+Once the change is made, make sure to reset your node for it to take effect. At the end of this guide you will remove this option if it is no longer needed.
+
+### Create a wallet
+
+In order to manage accounts, you first have to create a wallet to hold the seed or private keys for the accounts. If you started the node from the published binary builds or Docker images a wallet will have already been created. There will also be a cryptographically secure [seed](../integration-guides/the-basics.md#seed) and the first private key (derived from the seed at index `0`) added to the wallet.
+
+For the purposes of this guide we will proceed as if these didn't exist, which will be the case for self-built nodes. We will instead use various commands to create the wallet and seed, backup seed details and add accounts. But before these RPC commands can be called a configuration option is needed.
+
+#### Run command
+
+After this configuration change you can create a wallet using the ['wallet_create' RPC](../commands/rpc-protocol.md#wallet-create). During this call not only will a wallet be created, but a cryptographically secure [seed](../integration-guides/the-basics.md#seed) will also be created and added to the wallet. If you wish to use an existing seed instead of have one generated, make sure to include it using the optional `seed` parameter.
+
+=== "Test network"
+	**Request**
+	```bash
+	curl -d '{
+	  "action": "wallet_create"
+	}' http://127.0.0.1:17076
+	```
+
+=== "Main network"
+	**Request**
+	```bash
+	curl -d '{
+	  "action": "wallet_create"
+	}' http://127.0.0.1:7076
+	```
+
+=== "Beta network"
+	**Request**
+	```bash
+	curl -d '{
+	  "action": "wallet_create"
+	}' http://127.0.0.1:55000
+	```
```

Comment:
Why is the Main network only in the second place? Shouldn't it be the first tab as it's the most important? IMO it should be Main/Beta/Test
The same issue applies to the node-setup page too.

Reply:
Depends on the audience. In most cases, I would imagine it will be a developer who wants to integrate with the nano network. In which case, they will want to set up a test network node first.

Now if the audience is a holder or someone who wants to participate in consensus then main net is what they are looking for. 

Either way, people will find what they need since the tabs are right next to each other.

Reply:
The goal of the guide is to help those new to setting up a node learn how to do so, with minimal impacts to themselves and the network in general, so the test network seems like a good starting point for this. Once they are more comfortable the main network is just a tab click away for the details. Having the beta network be last was also intentional, since only the most involved and dedicated should be setting up on beta to help with node improvements (as opposed to the more common integration testing which is better done on the more stable test network).